src/website/views/templates/authentication.pug

```diff
@@ -27,21 +27,22 @@ h3 App Basic Sign In
             a.a-btn.-flat.-icon.-light(href="#")
               .a-btn__content
                 i.a-icon.icon-circle-question
-      .chi-app__body.-px-sm--2.-py-sm--4.-d--flex.-justify-content--center
-        .-pb--2.-w--100.-mw--480
-          .a-card.-b--0.-bg--none
-            .a-card__content.-text--center.-px--4.-px-sm--6.-pb--4
-              h3.-text--normal Sign In
-              .a-inputWrapper.-floatingLabel.-pt--2#aemail
-                input(type='email', class='a-input -xl', id='email')
-                label(for='email') Email
-              .a-inputWrapper.-floatingLabel.-pt--1#apassword
-                input(type='password', class='a-input -xl', id='password')
-                label(for='password') Password
-              .m-formGroup.-pt--3.-pt-sm--4.-justify-content--between.-align-items--center.-flex--column.-flex-sm--row
-                a.-text.-mb--2.-mb-sm--0(href="#") Forgot email or password?
-                button.a-btn.-primary.-xl Sign In
-          .-text.-text--center.-py--2 Don't have an account? <a href="#">Sign Up</a>
+      .chi-app__body.-px-sm--2.-py-sm--4
+        .a-grid.-pb--2.-justify-content--center
+          .a-col.-w-md--7
+            .a-card.-b--0.-bg--none
+              .a-card__content.-text--center.-px--4.-px-sm--6.-pb--4
+                h3.-text--normal Sign In
+                .a-inputWrapper.-floatingLabel.-pt--2#aemail
+                  input(type='email', class='a-input -xl', id='email')
+                  label(for='email') Email
+                .a-inputWrapper.-floatingLabel.-pt--1#apassword
+                  input(type='password', class='a-input -xl', id='password')
+                  label(for='password') Password
+                .m-formGroup.-pt--3.-pt-sm--4.-justify-content--between.-align-items--center.-flex--column.-flex-sm--row
+                  a.-text.-mb--2.-mb-sm--0(href="#") Forgot email or password?
+                  button.a-btn.-primary.-xl Sign In
```

Comment:
@davidojedacl please add .-fluid, .-justify-content--center, .a-col and .-w-md--3